pyClient/zeth/encryption.py

```diff
@@ -18,8 +18,64 @@
 from cryptography.hazmat.primitives import hashes, poly1305
 from cryptography.hazmat.primitives.serialization import Encoding, PrivateFormat,\
     PublicFormat, NoEncryption
+from cryptography.exceptions import InvalidSignature \
+    as cryptography_InvalidSignature
 
 
+# Expose the exception type
+InvalidSignature = cryptography_InvalidSignature
+
+
+# pylint: disable=line-too-long
+
+# References:
+#
+# \[Bernstein06]
+#  "Curve25519:new Diffie-Hellman speed records"
+#  Daniel J. Bernstein,
+#  International Workshop on Public Key Cryptography, 2006,
+#  <https://cr.yp.to/ecdh/curve25519-20060209.pdf>
+#
+# \[LangleyN18]
+#  "Chacha20 and poly1305 for ietf protocols."
+#  Adam Langley and Yoav Nir,
+#  RFC 8439, 2018,
+#  <https://tools.ietf.org/html/rfc8439>
+
+# This implementation makes use of the `cryptography` library with OpenSSL
```

Comment:
Thanks for double checking on that @dtebbs 
Can we change this comment to a module-level docstring in order to add the explanations below to the module documentation?

Reply:
Moved to docstring

Reply:
Thanks